openstudiocore/src/utilities/core/FileReference.cpp

```diff
@@ -150,7 +150,7 @@ void FileReference::setPath(const openstudio::path& newPath) {
 bool FileReference::makePathAbsolute(const openstudio::path& searchDirectory) {
```

Comment:
I'm curious about this FileReference class in general. I wonder who is using it. Can its function be rolled into the new openstudio::filesystem?